install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -109,6 +109,122 @@ key "update-key" {
+openshift_openstack_node_subnet_name: *<deployment-subnet-name>*
+openshift_openstack_router_name: *<deployment-router-name>*
+openshift_openstack_master_floating_ip: *false*
+openshift_openstack_infra_floating_ip: *false*
+openshift_openstack_compute_floating_ip: *false*
+## End of Optional Floating IP section
+
+openshift_openstack_num_masters: *3*
+openshift_openstack_num_infra: *3*
+openshift_openstack_num_cns: *0*
+openshift_openstack_num_nodes: *2*
+
+openshift_openstack_master_flavor: *"m1.master"*
+openshift_openstack_default_flavor: *"m1.node"*
+
+## Kuryr configuration
+openshift_use_kuryr: True
+openshift_use_openshift_sdn: False
+use_trunk_ports: True
+os_sdn_network_plugin_name: cni
+openshift_node_proxy_mode: userspace
+kuryr_openstack_pool_driver: nested
+openshift_kuryr_precreate_subports: 5
+
+kuryr_openstack_public_net_id: *<public_ID>*
+
+# To enable namespace isolation, uncomment
+#openshift_kuryr_subnet_driver: namespace
+#openshift_kuryr_sg_driver: namespace
+
+# Select kuryr image (always latest available)
+openshift_openstack_kuryr_controller_image: registry.access.redhat.com/rhosp14/openstack-kuryr-controller:latest
+openshift_openstack_kuryr_cni_image: registry.access.redhat.com/rhosp14/openstack-kuryr-cni:latest
+
+openshift_master_open_ports:
+- service: dns tcp
+  port: 53/tcp
+- service: dns udp
+  port: 53/udp
+openshift_node_open_ports:
+- service: dns tcp
+  port: 53/tcp
+- service: dns udp
+  port: 53/udp
+# End of Kuryr configuration
+
+openshift_openstack_use_lbaas_load_balancer: *true*
+
+openshift_openstack_docker_volume_size: "15"
+
+# # Roll-your-own DNS
+*openshift_openstack_external_nsupdate_keys:*
+  public:
+    *key_secret: '/alb8h0EAFWvb4i+CMA12w=='*
+    *key_name: "update-key"*
+    *key_algorithm: 'hmac-md5'*
+    *server: '<ip-of-DNS>'*
+  private:
+    *key_secret: '/alb8h0EAFWvb4i+CMA12w=='*
+    *key_name: "update-key"*
+    *key_algorithm: 'hmac-md5'*
+    *server: '<ip-of-DNS>'*
+
+ansible_user: openshift
+
+## cloud config
+openshift_openstack_disable_root: true
+openshift_openstack_user: openshift
+----
+
+[NOTE]
+====
+To enable namespace isolation, there is a need to set 'openshift_kuryr_subnet_driver'
+to 'namespace' so that a new Neutron subnet is created by kuryr for each
+namespace. It is also required to set 'openshift_kuryr_sg_driver' to 'namespace'
+to ensure the proper security groups are created and used to enforce isolation
+between the different namespaces.
+====
+
+[NOTE]
+====
+Regardless of the overcloud Red Hat OpenStack version, the kuryr images should
+be always set to the latest one supported. For instance, at the moment, the
+kuryr images should be from OSP 14, regardless of the overcloud being OSP 14
+or OSP 13. The rationale behind is that kuryr is just another workload on top of
+the overcloud, and this way it can better align with OpenShift new features.
+====
+
+[NOTE]
+====
+When using Kuryr SDN, network policies and nodeport services are not currently
```

Comment:
Suggestion:

Network policies and nodeport services are not supported when  Kuryr SDN is enabled.